### ISSUE TITLE
No database connection for the categories class

### ISSUE THREAD

**Issue by manuelkleinert** [dachcom-digital/pimcore-lucene-search#54]
I need categories from the Pimcore objects. Unfortunately I don't have access to the class category.

AppBundle\LuceneSearch\Services\Categories
```
    public function getCategories() : array
    {
        $categoryArray = [];

        array_push($categoryArray, ['id' => 1, 'label' => 'Blog']);
        array_push($categoryArray, ['id' => 2, 'label' => 'Support']);
        array_push($categoryArray, ['id' => 3, 'label' => 'Project']);

        $catList = new DataObject\....\Listing(); // -> ERROR (Fatal error: Uncaught Symfony\Component\Debug\Exception\FatalThrowableError: Call to a member function get() on null in /var/www/html/pimcore/lib/Pimcore/Db.php on line 51) 
        foreach ($catList as $cat) {
            array_push($categoryArray, ['id' => $cat->getId(), 'label' => $cat->getName()]);
        }

        return $categoryArray;
    }
```

**Comment by solverat** [comment]
hey @manuelkleinert fixed in 2.0.2!

**Comment by manuelkleinert** [comment]
Thank you very much @solverat
